proposals/stage-1/PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -18,100 +18,140 @@ Projects must be proposed via GitHub. Project proposals submitted to the _$FOUND
 
+The Impact Stage is for projects that have reached their growth goals and are now on a sustaining cycle of development, maintainence, and long-term support. These projects have reached a   
+
 **Examples**
 
+1. Projects that have publicly documented release cycles and plans for LTS.
+1. Projects that have themselves become platforms for other projects.
+1. Projects that are able to attract a healthy number of committers on the basis of its production usefulness (not simply 'developer popularity').
+1. Projects that have several, high-profile or well known end-user implementations.
+
 **Expectations**
 
+Impact Stage projects are expected to participate actively in CPC proceedings, and as such have a binding vote on CPC matters requiring a formal vote, such as the election of a CPC Director. They receive ongoing financial and marketing support from the Foundation, and are expected to cross promote the foundation along with their activities. 
+
 **Acceptance Criteria**
-To graduate from Sandbox or Incubating status, or for a new project to join as a Top-Level project, a project must meet the Incubating stage criteria plus:
 
- * Have committers from at least two organizations.
+To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
+
+ * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
+ * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
  * Adopt the _$FOUNDATION_ Code of Conduct.
- * Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
+ * Explicitly define a project governance and committer process. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
  * Have a public list of project adopters for at least the primary repo (e.g., ADOPTERS.md or logos on the project website).
- * Receive a supermajority vote from the CPC to move to Graduation stage. Projects can attempt to move directly from Sandbox to Top Level, if they can demonstrate sufficient maturity. 
+ * Other metrics as defined by the applying Project during the application process in cooperation with the CPC.
+ * Receive a supermajority vote from the CPC to move to Impact stage. Projects can move directly from At Large to Impact, if they can demonstrate sufficient maturity and have met all requirements. 
```

Comment:
What of impact projects that no longer fulfill the requirements?

Reply:
Then they could transition to At Large projects (or Emeritus, or I suppose to another Foundation). I don't think we can/should assume that projects will remain in one state for the entire time...